Lib/test/test_fcntl.py

```diff
@@ -190,6 +190,18 @@ def test_fcntl_f_getpath(self):
         res = fcntl.fcntl(self.f.fileno(), fcntl.F_GETPATH, bytes(len(expected)))
         self.assertEqual(expected, res)
 
+    @unittest.skipIf(not (hasattr(fcntl, "F_SETPIPE_SZ") and hasattr(fcntl, "F_GETPIPE_SZ")),
+                     "F_SETPIPE_SZ and F_GETPIPE_SZ are not available on all unix platforms.")
+    def test_fcntl_f_pipesize(self):
+        test_pipe_r, test_pipe_w = os.pipe()
+        pipesize = 16 * 1024  # 64K is linux default.
```

Comment:
minor nit:

rather than rely on an assumed default, use F_GETPIPE_SZ to determine the default.  then set the value to half or double that and confirm that it worked.

Reply:
Great suggestion! I was struggling how to determine the default myself. `os.sysconf` values do not help here unfortunately. This is much easier. I feel a bit stupid for having overlooked it.